spec/factories/user.rb

```diff
@@ -5,6 +5,7 @@
       role nil
       # e.g.: "miq_request_approval"
       features nil
+      tenant { Tenant.seed }
```

Comment:
I don't know anymore.

Not sure how the system is supposed to work with a user without a group or tenant.

you are right - this will be a hit. thanks. will mull over